ligo/stablecoin/spec.ligo

```diff
@@ -252,13 +302,16 @@ type ledger is big_map (address, nat)
 type minting_allowances is map (address, nat)
 
 type storage is record
-  ledger                      : ledger
-; token_metadata_registry     : address
-; minting_allowances          : minting_allowances
-; paused                      : bool
-; roles                       : roles
-; operators                   : operators
-; transferlist_contract       : option(address)
+  ledger                  : ledger
+; token_metadata_registry : address
+; minting_allowances      : minting_allowances
+; paused                  : bool
+; roles                   : roles
+; operators               : operators
+; transferlist_contract   : option(address)
+; counter                 : counter
```

Comment:
I would rename it to something like `permit_counter` because just `counter` sounds a bit too generic, imo. Or maybe I would define a separate data type holding all permit-related data.

Though, since renaming a field in LIGO changes data type structure and forces us to update Haskell code, I don't insist on doing that :) But at least adding a comment would be nice.

Reply:
Agreed, renamed to `permit_counter` :) I also think the `revoke`/`get_counter` entrypoints could be renamed to `revoke_permit`/`get_permit_counter` in the spec.

Reply:
It's something to add to TZIP-17 comments :)